staticfiles/templates/contributors/home.html

```diff
@@ -228,6 +228,13 @@ <h1 class="page-header">
               <span>Radhika Bhanu</span> 
             </a>
           </div>  
+          
+           <div class="col-sm-2 col-xs-4 thumb">
+            <a class="contributor-thumb" href="/contributors/saviaga">
+              <img class="thumbnail img-responsive" src="/static/images/contributors/saviaga.jpg" />
```

Comment:
1. saviaga.jpg is missing, 
2. could raise one PR instead of #171 #170. Thank you!